src/org/labkey/test/tests/wiki/WikiTest.java

```diff
@@ -149,6 +152,25 @@ public void testSteps()
         assertTextPresent("This folder does not currently contain any wiki pages to display");
     }
 
+    @Test
+    public void testShowPageTreeForWiki()
+    {
+        goToProjectHome();
+        log("Creating the wiki");
+        WikiHelper wikiHelper = new WikiHelper(this);
+        wikiHelper.createNewWikiPage("HTML");
+        setFormElement(Locator.name("name"), WIKI_PAGE_NAME);
+        setFormElement(Locator.name("title"), WIKI_PAGE_TITLE);
+        wikiHelper.setWikiBody(WIKI_CHECK_CONTENT);
+        wikiHelper.saveWikiPage();
```

Comment:
Use existing page class (`wiki.EditPage`)
```suggestion
        wikiHelper.createNewWikiPage("HTML")
            .setName(WIKI_PAGE_NAME)
            .setTitle(WIKI_PAGE_TITLE)
            .setBody(WIKI_CHECK_CONTENT)
            .saveAndClose();
```

Reply:
SetBody is not working as expected, so using setWikiBody from wikiHelper.